src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyState = { ...state };
+  const result = [];
+
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
Use a destruction for `action`
```suggestion
  for (const action of actions) {
    const { type, ....... } = action;
  
    switch (type) {
```